src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { RegisterData, AuthenticateData, APIAuthenticateResponse, APIUser, APIEvent, ProcessedAPIEvent, EventCreationData, EventEditData, GetMessageData, ProcessedAPIMessage, APIMessage, APIDMChannel, ProfileUploadData } from './types/api';
+import { RegisterData, AuthenticateData, APIAuthenticateResponse, APIUser, APIPrivateUser, APIEvent, ProcessedAPIEvent, EventCreationData, EventEditData, GetMessageData, ProcessedAPIMessage, APIMessage, APIDMChannel, ProfileUploadData } from './types/api';
```

Comment:
There are linter errors because `APIPrivateUser` is imported but not used. However, now that you've created this interface we can properly type `getMe()` which actually does return an `APIPrivateUser`. Could you update the signature of that method to reflect this?